metaci/cumulusci/logger.py

```diff
@@ -49,14 +52,20 @@ def init_logger(model):
 
     logger = logging.getLogger("cumulusci")
 
-    # Remove existing handlers
-    for handler in list(logger.handlers):
-        handler.stream.flush(force=True)
-        logger.removeHandler(handler)
+    if os.environ.get("LOG_TO_STDERR"):
+        handler = logging.StreamHandler()
+        print("Logging to STDERR")
+        model.log = "Logging to STDERR"
```

Comment:
I can see the use of this for debugging locally, but it should probably log to stderr _in addition_ to the db model, rather than instead of it.